src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -57,6 +65,14 @@ public function get($name)
         return parent::get($name);
     }
 
+    /**
+     * Gets the common env placeholder prefix for env vars created by this bag.
+     */
+    public function getEnvPlaceholderPrefix(): string
```

Comment:
here and also in Config, I'd suggest to name this `*UniquePrefix` - to hint that it's not only a prefix, but it's a very distinctive one